lib/aur-build

```diff
@@ -329,9 +339,8 @@ fi
 
 if (( chroot )); then
     # Update pacman and makepkg configuration for the chroot build
-    # queue. A full system upgrade is run on the /root container to
-    # avoid lenghty upgrades for makechrootpkg -u.
-    aur chroot "${chroot_args[@]}" --create --update
+    # queue.
```

Comment:
Was removing the second part of the comment intentional?

Reply:
since we're still using the old chroot I'm going to add it back